tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -93,7 +93,7 @@ def format_message(self, alert):
             suffix = "status changed"
         elif alert.alert_type == ALERT_TYPES[1]:
             suffix = "threshold breached"
-        return "Subject: [Tendrl Alert] %s, %s: %s\n\n%s" % (
+        return "Subject: [Alert] %s, %s: %s\n\n%s" % (
             resource, severity, suffix, message)
```

Comment:
We should be doing "Subject: [Alert] [%severity], resource: suffix \n \n message"

@brainfunked

Reply:
This is fine only if the 'from' address of the email configuration is set to tendrl@<hostname>.